Doc/install/index.rst

```diff
@@ -696,7 +696,7 @@ is supplied to suppress this behaviour.  So you could simply edit
    import sys
    sys.path.append('/www/python/')
 
-However, if you reinstall the same minor version of Python (perhaps when
+However, if you reinstall the same feature version of Python (perhaps when
```

Comment:
```suggestion
However, if you reinstall the same feature release of Python (perhaps when
```

Reply:
I might actually keep this one as "minor version" -- while I agree that "feature release/bugfix release" is a clearer terminology in general, in this specific case both "feature release" and "feature version" read more awkwardly than "minor version", in my opinion. But I'm curious for @CAM-Gerlach's thoughts.

Reply:
Yeah, I pondered about this myself. I'm not sure what reads best here. Perhaps we can phrase the sentence in a better way.

Reply:
"Reinstall" also feels weird here. I do not think of upgrading to a new patch/micro version as reinstalling.

Reply:
These docs are super-outdated, are the ancient legacy version of the newer `installing.rst` doc, and are scheduled for removal in 3.12 anyway as they are distutils-specific, which is being removed in 3.12 (I've just been waiting on confirmation of Setuptools plans/needs here to actually do the removal). I would definitely not bother touching them now (or spending more time thinking about this, heh).

```suggestion
However, if you reinstall the same minor version of Python (perhaps when
```

Reply:
SGTM